docs/spec_draft.md

```diff
@@ -1586,6 +1587,48 @@ specification. Numeric precision is implementation-defined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.select
+
+### Semantics
+
+Produces a `result` tensor where each element is selected from `on_true` or
+`on_false` tensor based on the value of the corresponding element of `pred`.
+More formally,
+`result[i0, ..., iR-1] = predicate[i0, ..., iR-1] ? on_true[i0, ..., iR-1] : on_false[i0, ..., iR-1]`,
+where `predicate[i0, ..., iR-1] = rank(pred) == 0 ? pred : pred[i0, ..., iR-1]`.
```

Comment:
One further simplification: rename `predicate[i0, ..., iR-1]` to just `predicate`. `[i0, ..., iR-1]` are already called out in `pred[i0, ..., iR-1]`, so there's no need to call them out in predicate, unless we genuinely want to create a broadcasted tensor and use it somewhere (which I don't think is the case here).

Reply:
Makes sense